m4/pbs_default_scheduler.m4

```diff
@@ -37,12 +37,7 @@
 
 AC_DEFUN([PBS_AC_DEFAULT_SCHEDULER],
 [
-  AC_MSG_CHECKING([for Default scheduler name])
```

Comment:
Does this need to be done in the configure process?  Can't it just be a #define in the code itself?